src/sentry/dynamic_sampling/tasks/recalibrate_orgs.py

```diff
@@ -46,7 +46,7 @@ def recalibrate_orgs(context: TaskContext) -> None:
     ):
         valid_orgs = []
         for org_volume in org_volumes:
-            if org_volume.is_valid_for_recalibration:
+            if org_volume.is_valid_for_recalibration():
```

Comment:
Crying in Python

Reply:
lawl

Reply:
Yeah, but I want to make it explicit that it is a computed result, instead of a property. I just messed up the syntax :(